src/PythonMigrationViewExtension/Docs/PythonMigrationWarning.html

```diff
@@ -207,18 +207,18 @@ <h3>How to make changes to your Python nodes for Python 3 </h3>
     <p>
         If you are curious about diving deeper into the differences between the two Python versions, here are two
         external links to get you started:
-    <ul>
-        <li>
-            <a href="https://docs.python.org/3/howto/pyporting.html">
-                The official "Porting Python 2 Code to Python
-                3"
-                guide
-            </a>
-        </li>
-        <li>
-            <a href="http://python3porting.com/">Supporting Python 3: An in-depth guide" guide</a>
-        </li>
-    </ul>
+        <ul>
+            <li>
+                <a href="https://docs.python.org/3/howto/pyporting.html">
+                    The official "Porting Python 2 Code to Python
+                    3"
+                    guide
```

Comment:
This was already
like
this
but we
should make
it one line
I guess.
:)

Reply:
Let me check the whole file, I agree